SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -346,4 +356,8 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	// before the actor holding the position for all the hierarchy, it can immediately attempt to migrate back.
 	// Using this timestamp, we can back off attempting migrations for a while.
 	uint64 AuthorityReceivedTimestamp;
+
+	// In case the actor's owner did not have an entity ID when trying to set interest to it
```

Comment:
When does this happen? Startup actors?

Reply:
I have not seen it happened, I thought it could, in case a character gets replicated before its PC, but PlayerControllers get their entity id very early when a player connects, by some implementation detail of the connection startup.
I think it could happen if for whatever reason a PlayerController is manually created outside of this flow ?
It's a "just-in-case" code.